tests/PriorityMapTest.php

```diff
@@ -89,20 +91,37 @@ public function testGetKeys()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(
+        $result = $list->getKeys();
+        $resultValues = array_values($result);
+
+        $this->assertCount(5, $result);
```

Comment:
You already check coun t of actual key list in assertSame

Reply:
You're right. The ```assertCount``` should be removed.